components/NavBar/AccountMenu.jsx

```diff
@@ -94,13 +94,6 @@ export default function AccountMenu() {
         <MenuItem>
           <Avatar /> My account
         </MenuItem>
-        <MenuItem>
-          <Avatar />
-          <Link to="/stats" style={{ 'text-decoration': 'none', color: 'black' }}>
-            {' '}
-            My Reading Stats{' '}
```

Comment:
:crying_cat_face: We're removing the stats from the NavBar?

Reply:
Proposing to remove it from the user icon dropdown menu and move to the navigation bar